data/clients.json

```diff
@@ -252,13 +252,15 @@
     },
     {
         "doap": null,
-        "last_renewed": "2019-07-16T20:02:35",
+        "last_renewed": "2021-01-12T00:00:00",
```

Comment:
It doesn't even support XMPP anymore. Let's remove it entirely please. It is also know as scammer/bloat software.